rest/handler.go

```diff
@@ -189,6 +189,10 @@ func (h *handler) invoke(method handlerMethod) error {
 
 	h.logRequestLine()
 
+	if base.LogKeys["HTTP++"] {
```

Comment:
I added `LogEnabledExcludingLogStar()` and changed it to use this in the places that looked unsafe